modules/dnn/test/test_caffe_importer.cpp

```diff
@@ -186,7 +194,7 @@ TEST_P(Reproducibility_MobileNet_SSD, Accuracy)
     Mat out = net.forward();
 
     Mat ref = blobFromNPY(_tf("mobilenet_ssd_caffe_out.npy"));
-    normAssertDetections(ref, out);
+    normAssertDetections(ref, out, "", 0.0, 4e-4, 5e-3);
```

Comment:
Please keep default values `1e-5` and `1e-4` for non-`DNN_TARGET_OPENCL_FP16` targets.

Reply:
fixed, thank you